mava/systems/jax/builder.py

```diff
@@ -13,28 +13,188 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# TODO (Arnu): reintroduce proper return types, e.g. data/parameter server once those
+# have been created.
+
 """Jax-based Mava system builder implementation."""
 
 from typing import Any, List
 
+from mava.callbacks import CallbackHookMixin
+from mava.core_jax import SystemBuilder
+
 
-class Builder:
+class Builder(SystemBuilder, CallbackHookMixin):
     def __init__(
         self,
         components: List[Any],
     ) -> None:
         """System building init
 
         Args:
-            components: system callback components
+            components: system callback blocks
```

Comment:
so blocks are analogous to components that make up a system? why do we need the blocks class exactly?

Reply:
Oops I think this is a type that creeped in with a global find and replace 😅

Reply:
Will fix it.